expression_data_matrix.md

```diff
@@ -40,6 +40,43 @@ The bundle metadata is intended to contain metadata common to all the samples an
 
 It may occur that the matrix contains samples from multiple projects or studies.  If elements like the processing pipeline are no longer common to all samples, care should be taken to move this metadata to the samples or features metadata table instead of the bundle metadata.
 
+# Sample Metadata Discussion
+
+### Minimum Sample Metadata
+
+The following metadata fields describe a recommended minimal set of sample metadata.
+
+<table>
+<tr markdown="block"><td>
+`sampleName`
+</td><td>
+_string_
+</td><td>
+A human readable name for the sample.  Would be better if it were unique to the sample within the database.
+</td></tr>
+<tr markdown="block"><td>
+`materialType`
+</td><td>
+_string_
+</td><td>
+The type and origin for the sample.  Should follow [MIABIS-2.0-14](https://github.com/MIABIS/miabis/wiki/Structured-data-and-lists#material-type) 
+</td></tr>
+<tr markdown="block"><td>
+`libraryPrep`
```

Comment:
Imagine a case where we return a matrix that half of the samples are from one study and other half from another study. In that case we cannot have a common bundle metadata (as discussed in #6). Shouldn't we move all the bundle metadata to the samples? For example we need to know for each sample what is the reference, annotation, unit,... .

I think we either:
 * don't need `libraryPrep` (since it's common for all samples and is stored in the bundle metadata).
 * `libraryPrep` will be all items mentioned in #6 for bundle metadata.

Reply:
What about modeling the bundle metadata as a collection of bundles/projects and adding a bundleID/projectID attribute to the sample metadata?

Reply:
This seems to be the best way to do it.  The individual samples can then reference a short ID rather than repeating what may be a large amount of data at that level

Reply:
Thanks @emi80, that sounds more elegant than what I suggested.
@saupchurch so I guess we need to unique bundle ID to #6.